sigs/addons/MIGRATION_TO_CORE.md

```diff
@@ -0,0 +1,32 @@
+# Migration From TF-Addons To TensorFlow Core
+
+### In-Progress & Previous Migrations:
+https://github.com/tensorflow/addons/projects/2/
```

Comment:
@theadactyl This may be more visible to a larger audience if the project is housed in tf/community

Reply:
It may be problematic since the Addons maintainers don't have write access to it, it may be possible that it's not updated often. If we have control over it, we can take over the responsability of updating it.

Reply:
@gabrieldemarmiesse I agree with you cause we had not a really good historic records for this repo (see https://github.com/tensorflow/community/issues/226).
I think that the upstreaming/promotion process will enter soon or later in "submarine mode".
I suppose this will happen just after we have surpassed all the requirements/thresholds on the SIGs side and so as soon as the RFC could be formally opened here cause it is complaint with the bare minimal template thresholds/constrains (I hope also quantitative) that we are trying to define together in this review. 
We will have not so much control over the Kanban states cause the process at some point will be almost internal to the TF/core team (internal evaluations, internal activities looking for a sponsors etc.) so the ownership will be of the TF/core team and we can just suppose here that we could have some discrete "emergent" info about this phases of the process that could impact an update of a Kanban states here and/or  RFC labels handling. 
So I suppose we could manage a simplified Kanban on our side that is limited to our ownership of the process and so the last state before the final accepted/rejected could be just something like  "RFC submitted to community repo". 
From this point we need to track the process here cause we are totally dependent from discrete/sparse info emerging from the TF internal process and so if and when the TF team is able to release new info I think it could also handle the ownership of the state in Kanban and labels update.
We could think differently if the process will be totally public so we are on par about all the info related to the process or SIG members could have some different role here in the repo but as I understand with the current repo polcies IMHO this is not the case (see https://github.com/tensorflow/community/issues/227)

Reply:
Agreed I think this is more sustainable if the Addons team is able to manage the project.